docs/examples/setup-symfony.rst

```diff
@@ -118,7 +117,7 @@ to its expected path.
 
 .. code-block:: bash
 
-   devilbox@php-7.0.20 in /shared/httpd/my-symfony $ ln -s symfony/web/ htdocs
+   devilbox@php-7.0.20 in /shared/httpd/my-symfony $ ln -s symfony/public/ htdocs
```

Comment:
Can you search and replace `7.0.20` to `7.2.29` as the current Symfony documentation says it requires at least PHP 7.2 currently

Reply:
Ok